doc/source/examples/multiphysics/laser-heating/laser-heating.rst

```diff
@@ -0,0 +1,231 @@
+
+    .. math:: 
+        q(x,y,z) = \frac{\eta \alpha P}{\pi r^2 \mu} \exp{(-\eta \frac{r^2}{R^2})} \exp{(- \frac{|z|}{\mu})}
+
+
+where :math:`\eta`, :math:`\alpha`, :math:`P`, :math:`R`, :math:`\mu`, :math:`r` and :math:`z` denote concentration factor, absorptivity, laser power, beam radius, penetration depth, radial distance from the laser focal point, and axial distance from the laser focal point, respectively. These parameters are explained in more detail in `laser parameters <https://lethe-cfd.github.io/lethe/parameters/cfd/laser_heat_source.html>`_.
+
+
+.. note:: 
+    The scanning path of the laser is defined using a Function expression in the ``path`` subsection. Here the laser ``path`` is a function of time, and changes its direction twice during laser operation.
+
+
+.. code-block:: text
+
+    #---------------------------------------------------
+    # Laser parameters
+    #---------------------------------------------------
+    subsection laser parameters
+        	set enable = true
+        	set concentration factor = 50
+        	set power = 3
+        	set absorptivity = 0.6
+        	set penetration depth = 0.00005
+        	set beam radius = 0.00005
+        	set start time = 0
+        	set end time = 0.003
+        	set beam orientation = z-
+        	subsection path
+        		set Function expression = if(t<0.001, 0.5 * t, if(t<0.002, 0.0005, if(t<0.003 , 0.0005-0.5 * (t-0.002), -1))); if(t<0.001, 0.00025, if(t < 0.002, 0.00025 - 0.5 * (t-0.001) , if(t < 0.003 , -0.00025, -1))) ; 0.0003
+        	end
+    end    
+
+
+In the ``mesh adaptation`` subsection, we choose a mesh refinement based on the variable ``temperature``.
+
+
+.. code-block:: text
+
+    #---------------------------------------------------
+    # Mesh Adaptation
+    #---------------------------------------------------
+    subsection mesh adaptation
+      set type                    = kelly
+      set variable                = temperature
+      set fraction type           = fraction
+      set max refinement level    = 4
+      set min refinement level    = 0
+      set frequency               = 1
+      set fraction refinement     = 0.5
+      set fraction coarsening     = 0.2
+    end
+
+----------------------
+Running the simulation
+----------------------
+
+Call the gls_navier_stokes_3d by invoking:  
+
+``mpirun -np 8 gls_navier_stokes_3d laser_heating.prm``
+
+to run the simulation using eight CPU cores. Feel free to use more.
+
+
+.. warning:: 
+    Make sure to compile lethe in `Release` mode and 
+    run in parallel using mpirun. This simulation takes
+    :math:`\approx` 5 minutes on 8 processes.
+
+
+
+-------
+Results
+-------
+
+The following animation shows the temperature distribution in the simulations domain, as well the laser path.
```

Comment:
I really love this example, it's so simple but nice.
I would add a possibility for extension where people could add the phase change physical property

Reply:
Added.